AEP/aep-00001.md

```diff
@@ -0,0 +1,191 @@
+The final implementation must include test code and documentation appropriate for both the ArmoniK Developers and Users.
+
+10. **Rejected Ideas** – Throughout the discussion of an AEP, various ideas will be proposed which are not accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the AEP as well as preventing people from bringing up the same rejected idea again in subsequent discussions.
+In a way this section can be thought of as a breakout section of the Rationale section that is focused specifically on why certain ideas were not ultimately pursued.
+
+11. **Open Issues** – While an AEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the AEP to be ready for consideration are complete and reduces people duplicating prior discussion.
+
+12. **Footnotes** – A collection of footnotes cited in the AEP, and a place to list non-inline hyperlink targets.
+
+13. **Copyright/license** – Each new AEP must be placed under a dual license of public domain and CC0-1.0-Universal (see this AEP for an example).
+
+## AEP Formats and Templates
+
+AEPs are UTF-8 encoded, english text files using the [GitHub-flavored markdown](https://github.github.com/gfm/) format. Using undocumented Markdown syntax which "works" at one point with GitHub is discouraged, as it could break later without notice. Further specifications can be found in the [GitHub documentation](https://docs.github.com/en/get-started/writing-on-github). The AEP must pass the linter executed via the following command at the root of the repository:
+
+```shell
+npm run lint:md
+```
+
+An AEP format guide and template is available in [AEP-00005](https://github.com/aneoconsulting/ArmoniK.Community/blob/main/AEP/aep-00005.md).
+
+## AEP Header Preamble
+
+Each AEP must begin with a Markdown heading and table using the following format:
+
+```markdown
+# AEP {AEP number} : {AEP Title}
+
+||ArmoniK Enhancement Proposal|
+---:|:---
+**AEP**| AEP number
+**Title**| AEP Title
+**Author**| Author name <<Author-email>>
+**Status**| (Draft | Active | Accepted | Provisional | Deferred | Rejected | Withdrawn | Final | Superseded)
+**Type**| (Standard | Informational | Process)
+**Requires**| AEP numbers
+**ArmoniK Version**| ArmoniK version
+**Replaces**| AEP number
+**Superseded-By**| AEP number
+**Creation Date**| Creation date of the AEP (YYYY-MM-DD format)
+```
+
+The "Requires", "ArmoniK Version", "Replaces" and "Superseded-By" fields can be omitted for AEP where they're not relevant.
+The Author field lists the names, and optionally the email addresses of all the authors/owners of the AEP. The format of the Author field values must be:
+
+```markdown
+Random J. User <<random@example.com>>
```

Comment:
whey two << and >>? One is enough.

Reply:
Because in github flavored markdown, <> is considered a link. So we need double brackets to have actual brakets. See example : https://github.github.com/gfm/#example-170